cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsRdsDbParameterGroupView.java

```diff
@@ -9,6 +9,8 @@
 
 public class AwsRdsDbParameterGroupView {
 
+    public static final String DEFAULT_FAMILY_VERSION = "10";
```

Comment:
Perhaps could be reduced to package private if no other usage in cloud-aws is desired.

Reply:
it is used by the test. I annotated now as the other field